test/test_multicohort.py

```diff
@@ -184,26 +183,18 @@ def mock_get_sgs_by_cohort_project_b(*args, **kwargs) -> list[dict]:
     ]
 
 
-def mock_get_cohorts(*args, **kwargs) -> dict:
+def mock_get_cohorts(cohort_id: str, *args, **kwargs) -> list[dict]:
```

Comment:
As mentioned previously, the SGs from metamist process has been simplified from 3 calls to 1. Now this mock is parametrised to return a specific list of dictionaries depending on which cohort ID was given as an argument